### PR TITLE
Edited file for class

### PR DESCRIPTION
**Pull request by dumforjh**

### CODE REVIEW

**Comment by dumforjh** [comment]
Did this for class

File: _data/semesters/Spring 22/dumforjh.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      "introduction": "I am excited to learn about all of you.",
+      "achievements": [
+          "achievement1",
+          "achievement2"
+      ],
+      "links": {
+          "site": site: "https://it3049c-students.github.io/1-online-portfolio-dumforjh/",
+          "linkedin": site: "https://www.linkedin.com/in/john-dumford-27a108201/",
+          "facebook": site: "https://www.facebook.com/profile.php?id=100012598806765",
+          "twitter": site: "https://twitter.com/john_dumford",
```
Comment (Ygilany) [inline _data/semesters/Spring 22/dumforjh.json:13]:
```suggestion
          "site": "https://it3049c-students.github.io/1-online-portfolio-dumforjh/",
          "linkedin":"https://www.linkedin.com/in/john-dumford-27a108201/",
          "facebook": "https://www.facebook.com/profile.php?id=100012598806765",
          "twitter": "https://twitter.com/john_dumford"
```

**Review by Ygilany — CHANGES_REQUESTED** [review]
You had this typo here

**Comment by Ygilany** [comment]
Please fill out the self-evaluation and make sure the assignment pass the automated checks

**Pull request closed without merging**